src/App.js

```diff
@@ -52,13 +52,14 @@ export const App = props => {
   } else {
     selectedTab = retrieveSelectedTab+distinguisher
   }
+  selectedTab = selectedTab.replace(/([^a-z0-9]+)/gi, "-")
```

Comment:
@mpolidori this regex `/([^a-z0-9]+)/gi` appears in three places, maybe create a variable for it? so that if later changes need to be made people wouldn't forget about the other two.

Also, the regex itself probably can be shortened for better readability. Do you want to match all non-word characters? If so, you can use `/\W+/gi`

Reply:
Ah, that's a much better regex! I made the change and updated the package versions in `package.json` for `datapackage-views-js`, `map-builder`, and `chart-builder`. Let me know what you think